tests/catch.hpp

```diff
@@ -1,13 +1,8 @@
-/*
- *  Catch v2.13.9
- *  Generated: 2022-04-12 22:37:23.260201
- *  ----------------------------------------------------------
- *  This file has been merged from multiple headers. Please don't edit it directly
- *  Copyright (c) 2022 Two Blue Cubes Ltd. All rights reserved.
- *
- *  Distributed under the Boost Software License, Version 1.0. (See accompanying
- *  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
- */
+// Copyright (c) 2022 Two Blue Cubes Ltd. All rights reserved.
+// SPDX-FileCopyrightText: 2022 stenc authors
+//
+// SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
Are you sure about this? I think it should be BSL-1.0

Reply:
definitively. Fixed that. Thank you.